Algorithms/Divide-and-Conquer/weighted_job_scheduling.cpp

```diff
@@ -0,0 +1,80 @@
+// Name : Shivam Singhal
+// Email ID : Shivam Singhal
+
+// Algorithm on weighted job scheduling
+
+#include <bits/stdc++.h>
```

Comment:
Follow
[Coding Standards](  https://github.com/sahilbansal17/Competitive_Coding/wiki/Coding-Standards)

Reply:
You can't use #include<bits/stdc++.h>